docs/faq.md

```diff
@@ -213,3 +213,48 @@ No. After changing the serviceType, AKO has to be rebooted and all objects which
 #### If serviceType is changed from NodePortLocal, would AKO remove NPL annotation from the Services automatically ?
 
 No. After changing the serviceType, the users have to remove NPL annotation from the Services themselves.
+
+#### Where can I see markers associated with Avi object?
```

Comment:
How can I see the marker labels associated with an object on the Avi Controller?